python/pyspark/pandas/data_type_ops/num_ops.py

```diff
@@ -395,3 +400,17 @@ def astype(
             return _as_string_type(index_ops, dtype, null_str=str(np.nan))
         else:
             return _as_other_type(index_ops, dtype, spark_type)
+
+
+class DecimalOps(FractionalOps):
+    """
+    The class for decimal operations of pandas-on-Spark objects with spark type:
+    DecimalType.
+    """
+
+    @property
+    def pretty_name(self) -> str:
+        return "decimal"
+
+    def isnull(self, index_ops: Union["Index", "Series"]) -> Union["Series", "Index"]:
```

Comment:
A second thought, if `isnull` returns `pyspark.sql.column.Column` (`index_ops.spark.column.isNull() | F.isnan(index_ops.spark.column)` in this case), https://github.com/apache/spark/blob/master/python/pyspark/pandas/data_type_ops/num_ops.py#L386-L389 can leverage `isnull` as well. How do you think about that?

Reply:
I think this `isnull` is designed for IndexOpsMixin to match the pandas's `isnull` API, so I think maybe it's not a good idea to make this `isnull` to cover col.isnull case.

The problem is that index_ops.spark.column is not datatype-based, if it's just for avoiding duplicated code, maybe we could add some utils function to address it.

Reply:
Sounds good.